### ISSUE TITLE
Add "Lanes" category

### ISSUE THREAD

**Issue by kolgza** [plepe/OpenStreetBrowser#119]
**Suggestion:** add a "Lanes" category, maybe under the "Individual Traffic" subcategory.

This would include how many lanes each road has and/or what each lane is used for.

https://wiki.openstreetmap.org/wiki/Key:lanes

**Comment by plepe** [comment]
This a very good idea. I started to create in a testing repository. I will show you soon :-)

**Comment by plepe** [comment]
Here you can try the lanes category - it's not finished yet and therefore not in the main repository:
https://www.openstreetbrowser.org/#categories=plepe/test/lanes

Lanes are visible from zoom level 17. What do you think?

**Comment by IgorEliezer** [comment]
Marvelous! :o

The orange lane is for parking, isn't it?

**Comment by plepe** [comment]
Yes, orange is parking. I would prefer to have a single color for parking lanes and just modify a symbol (or maybe variants of the color) to highlight restrictions. Also, I would like to show whether it's parallel, diagonal, ... parking.

Currently, cycle lanes are dark green, I thought about changing this to greyish red (like Netherlands cycle ways).

Sidewalks are missing. Turn lanes are not shown yet (I guess this is not so easy to implement currently, so this has to wait).

**Comment by IgorEliezer** [comment]
> Yes, orange is parking. I would prefer to have a single color for parking lanes and just modify a symbol (or maybe variants of the color) to highlight restrictions.

Something like this:
* no parking: ⦸⦸⦸⦸⦸ (https://www.compart.com/en/unicode/U+29B8)
* no stopping: ⨂⨂⨂⨂⨂ (https://www.compart.com/en/unicode/U+2A02)
